docs/one-click-integrations/jira-integration.md

```diff
@@ -0,0 +1,42 @@
+# Jira integration
+
+Pulse integrates with Jira to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+
+## Setting up the Jira integration
+
+To set up the Jira integration:
+
+1.  On Pulse, [expand **Integrations** and select **Jira**](https://app.pulse.codacy.com/integrations/jira){: target="_blank"}.<!--TODO Validate final link-->
+
+1.  Click **Install Jira** and follow the instructions on the Jira UI to install the app.
+
+    ![Installing the Pulse Jira app](images/jira-installing.png)
+
+1.  Wait until you get a confirmation that Pulse successfully created the integration and the necessary webhooks on Jira.
+
+    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+
+## Collected data
+
+The table below lists the data that the Jira integration collects from your Jira issues, together with:
+
+-   The mapping between the data collected from Jira and the [Pulse data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event)<!--TODO Check if API will have schema for Jira data model-->
+-   The metrics that Pulse calculates from the data to display on the dashboards
```

Comment:
@afsmeira @pedronetocodacy for the previous integrations we included a section like this listing the exact information that we were collecting from the third-party platforms, and how they mapped to our API data model (here are the pages for [GitHub](https://docs.pulse.codacy.com/one-click-integrations/github-integration/#collected-data) and [PagerDuty](https://docs.pulse.codacy.com/one-click-integrations/pagerduty-integration/#collected-data)).

Does it make sense to include similar information for Jira? In a way, this can help customers understand how their Jira setup and data updates can influence the calculation of the metrics on the Pulse side of things.

Reply:
We can't have exactly the same format since Jira data does not map to Pulse's data model (deployments, changes, incidents).

Here is my proposal:

```
<table>
<thead>
<tr>
<th><strong>Data collected from Jira</strong></th>
<th><strong>Details</strong></th>
<th><strong>Used in</strong></th>
</tr>
</thead>
<tbody>
    <tr>
        <td>- Issue</td>
        <td>
            Issue data includes all status transitions. For the purposes of Cycle Time computation, 'In Progress' corresponds
            to any Jira issue status that maps to the 'In Progress' status category, represented by the blue color in Jira.
            Jira issue status that correspond to the 'To Do' status category (represented by the grey color in Jira) do not
            contribute to cycle time.
        </td>
        <td>Lead time and cycle time on the <a href="../../metrics/lead-cycle-time/">Lead & Cycle time dashboard</a></td>
    </tr>
    <tr>
        <td>- Projects</td>
        <td>
            - N/A
        </td>
        <td>Filters for the <a href="../../metrics/lead-cycle-time/">Lead & Cycle time dashboard</a></td>
    </tr>
    <tr>
        <td>- Issue types</td>
        <td>
            - Issue types are grouped per name, since in Jira you can have the same issue type name in different projects.
            This means that different issue types named 'Subtask' and 'Sub-task' will be grouped in Pulse's UI and displayed under the name
            of the issue type that was processed first. For all effects, these issue types are the same for Pulse.
        </td>
        <td>Filters for the <a href="../../metrics/lead-cycle-time/">Lead & Cycle time dashboard</a></td>
    </tr>
</table>
```

Reply:
Thanks @afsmeira, these are some nice details to include!

I edited the text to make the information more straightforward and included the details "Project data includes the name of the Jira projects" for the Jira projects. Please have a look. :+1: